uniffi_build/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "uniffi_build"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Technically it's a `0.1.1` since there's not breaking changes in `uniffi_build`, but I think it is easier to reason about this by aligning versions.